h/claim/util.py

```diff
@@ -0,0 +1,12 @@
+def generate_claim_token(request, userid):
+    return request.registry.claim_serializer.dumps({'userid': userid})
```

Comment:
Ok, will do. FWIW I think this was just Aron being [consistent with the way `h.notification` does this at the moment](https://github.com/hypothesis/h/blob/a490e9fea71119304acc54abb77c8b6a28bda9e6/h/notification/__init__.py).